### ISSUE TITLE
I can't find "Send with USB" on Amazon for docs

### ISSUE THREAD

**Issue by johnl800** [apprenticeharper/DeDRM_tools#2067]
today I went to strip DRM from a document that was sent to my Amazon kindle. At the Amazon site cloud under "manage your content and devices" he "send to USB" option is no longer available for docs, only for books. what's up?

**Comment by ElleKayEm** [comment]
No, that's not an option for send to kindle docs.  You should be able to decypt the file directly from an E-ink Kindle.
